.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "plugins": ["prettier", "@amplitude/eslint-plugin-amplitude"],
+  "env": { "es6": true, "browser": true, "node": true, "mocha": true },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2018
+  },
+  "rules": {
+    "prettier/prettier": "error",
+    "no-prototype-builtins": "off"
+  },
+  "globals": {
+    "BUILD_COMPAT_REACT_NATIVE": "readonly",
+    "BUILD_COMPAT_LOCAL_STORAGE": "readonly",
+    "BUILD_COMPAT_SNIPPET": "readonly",
+    "BUILD_COMPAT_2_0": "readonly",
+    "assert": "readonly",
+    "expect": "readonly",
+    "should": "readonly",
```

Comment:
couldn't find a well maintained chai plugin, and the ones I found were essentially doing the exact same as what i'm doing here